src/Autonomous.cpp

```diff
@@ -412,30 +416,39 @@ void Autonomous::autonomyIter()
       map.addPoints(toTransform(pose), lidar_scan, mapDoesOverlap ? 0.4 : 0);
       mapDoesOverlap = lidar_scan.size() > mapOverlapSampleThreshold;
     }
-    if (plan_cost >= INFINITE_COST || ++time_since_plan % REPLAN_PERIOD == 0) {
-      // TODO planning should happen in a separate thread
-      // because it takes a long time
-      point_t point_t_goal;
-      point_t_goal.topRows(2) = plan_target.topRows(2);
-      point_t_goal(2) = 1.0;
-      point_t_goal = toTransform(pose) * point_t_goal;
-      //plan_t new_plan = getPlan(lidar_scan, point_t_goal, PLANNING_GOAL_REGION_RADIUS);
```

Comment:
I'd like to keep this commented-out line to make it easy to revert the map construction next week (hopefully this is not necessary, but just in case)